operator/internal/common/labels/labels.go

```diff
@@ -45,6 +45,10 @@ func WithDefaults(labels map[string]string) (map[string]string, bool) {
 	return newLabels, updated
 }
 
+func GetTlsSecretLabels() map[string]string {
```

Comment:
Can you move this function next to `commonLabels.DefaultLabels()`, and perhaps rename it to `TLSSecretLabels` for consistency?